Source/GSXib5KeyedUnarchiver.m

```diff
@@ -2725,6 +2728,19 @@ - (id) decodeToolbarImageForElement: (GSXibElement*)element
   return [self findResourceWithName: name];
 }
 
+- (id) decodeControlContentsForElement: (GSXibElement *)element
+{
+  NSNumber *num = [NSNumber numberWithInteger: 0];
```

Comment:
We will most likely need to implement variants here for other NSControl subclasses. But for now this should do.